uvm/impl/com/untangle/uvm/EventManagerImpl.java

```diff
@@ -1505,8 +1510,10 @@ private static boolean sendEmailForEvent( AlertRule rule, LogEvent event )
          * Report users
          */
         App reportsApp = UvmContextFactory.context().appManager().app("reports");
-        List<String> reportsEmailAddresses = ((Reporting) reportsApp).getAlertEmailAddresses();
-        alertRecipients.addAll(reportsEmailAddresses);
+        if(reportsApp == null) {
```

Comment:
Is this correct ? -> Add all `reportsEmailAddresses` in `alertRecipients` only if `reports` app is not present ?